cves/kernel/CVE-2021-22555.yml

```diff
@@ -482,7 +469,24 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Fantastic entry on your findings. I think this summarizes all that you found nicely. Good job!